Failing tests - Wrong time.Time parsing error string escaping

With the go version 1.15 the tests for the jsonData struct are failing.

```
go test ./...
--- FAIL: TestTransformation (0.00s)
    --- FAIL: TestTransformation/should_wrap_transforming_iso_date_error (0.00s)
        jsonData_test.go:32: 
                Error Trace:    jsonData_test.go:32
                Error:          Not equal: 
                                expected: "transforming jsonData to Data: parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: xxx"
                                actual  : "transforming jsonData to Data: parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: \"xxx\""
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -transforming jsonData to Data: parsing the json Date string produced an error: parsing time "2020-08-19T15:18:06+02:00xxx": extra text: xxx
                                +transforming jsonData to Data: parsing the json Date string produced an error: parsing time "2020-08-19T15:18:06+02:00xxx": extra text: "xxx"
                Test:           TestTransformation/should_wrap_transforming_iso_date_error
--- FAIL: TestParseISODateString (0.00s)
    --- FAIL: TestParseISODateString/should_return_error_on_wrong_string (0.00s)
        jsonData_test.go:108: 
                Error Trace:    jsonData_test.go:108
                Error:          Not equal: 
                                expected: "parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: xxx"
                                actual  : "parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: \"xxx\""
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -parsing the json Date string produced an error: parsing time "2020-08-19T15:18:06+02:00xxx": extra text: xxx
                                +parsing the json Date string produced an error: parsing time "2020-08-19T15:18:06+02:00xxx": extra text: "xxx"
                Test:           TestParseISODateString/should_return_error_on_wrong_string
FAIL
FAIL    github.com/hamburghammer/gmon/stats     0.004s
FAIL
```

--- a/stats/jsonData_test.go
+++ b/stats/jsonData_test.go
@@ -26,7 +26,7 @@ func TestTransformation(t *testing.T) {
 		wrongDateJSONData := getJSONData()
 		wrongDateJSONData.Date = "2020-08-19T15:18:06+02:00xxx"
 
-		want := "transforming jsonData to Data: parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: xxx"
+		want := "transforming jsonData to Data: parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: \"xxx\""
 		_, got := wrongDateJSONData.transformToData()
 
 		assert.Equal(t, want, got.Error())
@@ -101,7 +101,7 @@ func TestParseISODateString(t *testing.T) {
 	t.Run("should return error on wrong string", func(t *testing.T) {
 		currentTime := "2020-08-19T15:18:06+02:00xxx"
 
-		want := "parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: xxx"
+		want := "parsing the json Date string produced an error: parsing time \"2020-08-19T15:18:06+02:00xxx\": extra text: \"xxx\""
 		_, err := jsonData{Date: currentTime}.parseDateToTime()
 
 		assert.NotNil(t, err)

